README.md

```diff
@@ -216,14 +220,14 @@ onLogout(() => {
 
 ## Vue Router Guards
 
-If you're using Vue Router, you'll often need to guard some routes depending from the user authentication status.
+If you are using Vue Router, you will often need to guard some routes depending on the user authentication status.
 This package provides you some helpers to deal with common scenarios.
 
 ### `authGuard`
 
 Use this helper to create guards relying on the user authentication status.
-The first param your callback receives is a boolean representing the authentication status, while second and third params are the deprecated `to` and `from` Vue Router guards params.
-The returned guard is async as it awaits for the Auth0 client to be initialized before proceeding.
+The first param your callback receives is a boolean representing the authentication status, while the second and third params are the deprecated `to` and `from` Vue Router guards params.
+The returned guard is `async` as it `awaits` for the Auth0 client to be initialized before proceeding.
```

Comment:
Here `async` and `awaits` are used as an adjective and a verb, not as JS keywords

Reply:
Ok I have checked this. Currently async is not properly and English word. Rather it's the contraction of asynchronous, ofter used in tech environment. I mean, it's technically fine to use it like this, since we _are_ in a tech env, but it's not correct english.
About the await instead you are right.

Reply:
"async", as "sync", are part of commonly written and spoken english, at least into tech field, so... 🤷🏻‍♂️